### PR TITLE
Platform Celestica Seastone

### PR DESCRIPTION
**Pull request by spitchai**
```
Celestica seastone without ONLP
```

### CODE REVIEW

**Comment by spitchai** [comment]
@sonoble

File: packages/platforms/celestica/x86-64/x86-64-cel-seastone/onlp/PKG.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+variables:
+  platform: x86-64-cel_seastone-r0
```
Comment (jnealtowns) [inline packages/platforms/celestica/x86-64/x86-64-cel-seastone/onlp/PKG.yml]: ONL platform identifiers do not have underscores in them. This should be x86-64-cel-seastone-r0

File: packages/platforms/celestica/x86-64/x86-64-cel-seastone/onlp/PKG.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  install: /lib/platform-config/${platform}/onl
+
+common:
+  version: 1.0.0
+  arch: amd64
+  copyright: Copyright 2013, 2014, 2015 Big Switch Networks
+  maintainer: support@bigswitch.com
+  comment: dummy package for ONLP on Wedge
```
Comment (spitchai) [inline packages/platforms/celestica/x86-64/x86-64-cel-seastone/onlp/PKG.yml]: that is correct.

Comment (spitchai) [inline packages/platforms/celestica/x86-64/x86-64-cel-seastone/onlp/PKG.yml]: I will be submitting the seastone ONLP implementation in the next commit.

**Comment by jnealtowns** [comment]
@spitchai  I have merged these changes and updated them as per the latest platform requirements. 
See the following commits:
https://github.com/opencomputeproject/OpenNetworkLinux/commit/5d30997ce097f3e676273f815ba358bcc1b55e29
https://github.com/opencomputeproject/OpenNetworkLinux/commit/106a4029bcf64daa0b3bd9057a6610c02f7d54f1

**Pull request merged**